src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -16,20 +16,18 @@
 package edu.usf.cutr.gtfs_realtime.bullrunner;
 
 
-import java.util.List;
+import java.io.InputStream;
+import java.net.SocketTimeoutException;
+import java.time.Instant;

```

Comment:
When I import the project into IntelliJ it can't find this class because we're still referencing Java 1.7 in the pom.xml.  Could you please bump these to 1.8?

~~~
            <configuration>
                <source>1.8</source>
                <target>1.8</target>
		<encoding>UTF-8</encoding>
            </configuration>
~~~